### PR TITLE
feat(asana): add in typing for tasks (getTasksForSection)

### PR DESCRIPTION
**Pull request by laughnan**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
  - https://developers.asana.com/docs/get-tasks-from-a-section
  - https://github.com/Asana/node-asana/blob/master/lib/resources/gen/tasks.js#L314-L333
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/asana/index.d.ts [hunk 1]
```diff
@@ -2211,6 +2211,25 @@ declare namespace asana {
              * @return {Promise} The requested resource
              */
             updateTask(taskGid: string, data?: Tasks.UpdateParams, dispatchOptions?: any): Promise<Tasks.Type>;
+
+            // https://developers.asana.com/docs/get-tasks-from-a-section
+            // https://github.com/Asana/node-asana/blob/master/lib/resources/gen/tasks.js#L314-L333
+            /**
+             * * Get tasks from a section
+             *   * @param {String} sectionGid: (required) The globally unique identifier for the section.
+             *   * @param {Object} params: Parameters for the request
```
Comment (peterblazejewicz) [inline types/asana/index.d.ts:2220]:
```suggestion
             *   * @param {Object} [params]: Parameters for the request
```
keep consistent, e.g with `dispatchOptions` as optional param

File: types/asana/index.d.ts [hunk 2]
```diff
@@ -2211,6 +2211,25 @@ declare namespace asana {
...
+             *   * @param {Object} [dispatchOptions]: Options, if any, to pass the dispatcher for the request
+             *   * @return {Promise} The requested resource
+             * @param sectionGid
+             * @param params?
+             * @param dispatchOptions?
+             * @return
+             */
+            getTasksForSection(
```
Comment (peterblazejewicz) [inline types/asana/index.d.ts:2228]: can you please fix the JSDoc comment for this method? (duplicated lines, etc)

File: types/asana/index.d.ts (hunk 1 above)
Comment (laughnan) [reply types/asana/index.d.ts]: Updated in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62476/commits/c014532ffb4b3632937b4a4aed42da747d638d94

File: types/asana/index.d.ts (hunk 2 above)
Comment (laughnan) [reply types/asana/index.d.ts:2224]: Updated in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62476/commits/c014532ffb4b3632937b4a4aed42da747d638d94

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
those dispatch options seems to be not very well documented in the api (if not at all, that is in JS). That's ok for now though to use 'any'
@laughnan thanks!

**Comment by laughnan** [comment]
Thanks for the review @peterblazejewicz! I appreciate you taking the time.

**Comment by laughnan** [comment]
Ready to merge

**Pull request merged**